sandbox/src/game.ts

```diff
@@ -433,20 +433,39 @@ game.add(emitter);
 //emitter.follow(player, 20);
 
 var exploding = false;
-var trigger = new ex.Trigger(400, 200, 100, 100, () => {
-   if (!exploding) {
-      exploding = true;
-      emitter.isEmitting = true;
-      camera.shake(10, 10, 2000);
-      game.addTimer(new ex.Timer(() => {
-         emitter.isEmitting = false;
-         exploding = false;
-      }, 2000));
+// var trigger = new ex.Trigger(400, 200, 100, 100, () => {
```

Comment:
Did you intend to keep this for reference?

Reply:
Yah, I wanted to make sure it behaved the same. I can remove this if you want